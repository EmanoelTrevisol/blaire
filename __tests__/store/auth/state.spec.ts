import initialState from '@store/auth/state';

describe('store/auth/state', () => {
  test('matches snapshot', () => {
    expect(initialState).toMatchSnapshot();
  });
});
