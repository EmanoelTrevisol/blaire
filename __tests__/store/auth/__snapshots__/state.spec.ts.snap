// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`store/auth/state matches snapshot 1`] = `
Object {
  "user": null,
}
`;
